app/models/user.rb

```diff
@@ -4,5 +4,6 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable,
          :confirmable, :lockable, :trackable
+         INTEREST_FIELD =["Wedding", "Events", "Potraits"]
```

Comment:
single quotes  and alignment :)

Reply:
Add a presence validation for username and age.

Reply:
Done

Reply:
> Add a presence validation for username and age.

Done